sdk/react/src/DVCProvider.tsx

```diff
@@ -12,10 +12,14 @@ type Props = {
 let client: DVCClient | undefined
 
 export default function DVCProvider(props: Props): React.ReactElement {
-    const { envKey, user, options } = props.config
+    const { sdkKey, envKey, user, options } = props.config
+    const key = sdkKey || envKey
```

Comment:
@ajwootto updated this to allow for either `sdkKey` or `envKey`. Not crazy about how it means both need to be optional in the type def.

Reply:
posted a better type definition below

Reply:
How do I use that type? 
<img width="1202" alt="image" src="https://user-images.githubusercontent.com/1219069/216742239-75197480-3012-4b07-a8e5-736c32911335.png">